math/mathmore/src/GSLMultiRootFinder.cxx

```diff
@@ -158,7 +158,7 @@ const double * GSLMultiRootFinder::FVal() const {
 }
 const char * GSLMultiRootFinder::Name() const {
    // get GSL name
-   return (fSolver != 0) ? fSolver->Name().c_str() : "";
+   return (fSolver != 0) ? fSolver->Name().c_str() : nullptr;
```

Comment:
It's GCC 7 complaining. The warning in CDash can be found [here](http://cdash.cern.ch/viewBuildError.php?type=1&buildid=362082).